robot_sf/gym_env/robot_env.py

```
@@ -48,12 +49,15 @@ class RobotEnv(Env):
     """
 
     def __init__(
-            self,
-            env_config: EnvSettings = EnvSettings(),
-            reward_func: Callable[[dict], float] = simple_reward,
-            debug: bool = False,
-            recording_enabled: bool = False
-            ):
+        self,
+        env_config: EnvSettings = EnvSettings(),
```

Ruff

Do not perform function call `EnvSettings` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)